automation/build.sbt

```diff
@@ -19,21 +18,21 @@ cancelable in Global := true
 
 /**
   * sbt forking jvm -- sbt provides 2 testing modes: forked vs not forked.
-  * -- forked: each task (test class) is executed sequentially in a forked JVM.
+  * -- forked: each task (test class) is executed sequentially in a forked jvm.
```

Comment:
why?

Reply:
reworded to this
`-- forked: each task (test class) runs in a forked jvm. Tests within a task run sequentially by default.`

Reply:
I mean why the lowercasing?  Capital JVM makes more sense to me

Reply:
oh. no good reason. I'll change it back to uppercase.